share/completions/git.fish

```diff
@@ -1435,10 +1445,15 @@ complete -f -c git -n '__fish_git_using_command rebase' -l no-ff -d 'No fast-for
 # This actually takes script for $SHELL, but completing that is... complicated.
 complete -r -c git -n '__fish_git_using_command rebase' -l exec -d 'Execute shellscript'
 
+### reflog
```

Comment:
Maybe we want to eventually support reflog's subcommands (other than the default `show`), but `show` probably accounts for a large majority of uses so I think this is fine.

Reply:
Good idea. I added completions for the subcommands in 1f28825c276224a8752aacef1a548c2a32db65c4